docs/support/faq/why-cant-i-connect-to-my-vm-in-pouta.md

```diff
@@ -1,5 +1,107 @@
+
+Please follow the [Firewalls and security](/cloud/pouta/launch-vm-from-web-gui/#firewalls-and-security-groups) groups article. If the problem persists you may check the firewall setup of your local institution.
+
+### REMOTE HOST IDENTIFICATION HAS CHANGED
+
+Sometimes Floating IPs are reused with different Virtual Machines at different times. By default your SSH client will have `stricthostkeychecking=yes` configured, and will show you the error message:
+
+```sh
+$ ssh cloud-user@86.50.xxx.xxx
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+@    WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!     @
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+IT IS POSSIBLE THAT SOMEONE IS DOING SOMETHING NASTY!
+Someone could be eavesdropping on you right now (man-in-the-middle attack)!
+It is also possible that a host key has just been changed.
+The fingerprint for the ECDSA key sent by the remote host is
+SHA256:JURkzITHXHGavwz6fAahou5g4ii1q9CVuzLyImH5+tI.
+Please contact your system administrator.
+Add correct host key in /home/yyyy/.ssh/known_hosts to get rid of this message.
+Offending ECDSA key in /home/yyyy/.ssh/known_hosts:28
+  remove with:
+  ssh-keygen -f "/home/yyyy/.ssh/known_hosts" -R "86.50.xxx.xxx"
+ECDSA host key for 86.50.xxx.xxx has changed and you have requested strict checking.
+Host key verification failed.
+```
+
+You can safely do as it suggests and remove the entry. But only if you are sure that it is the first time you connect to said IP since it has been assigned to a new instance, or since the instance has been reinstalled. The example shows the output of the ssh command, different tools and versions might have a slightly different output, but the principle is the same.
+
+## Authentication
+
+```sh
+Received disconnect from xx.xx.xxx.xx port 22:2: Too many authentication failures
+```
+
+If the authentication is not confired properly, you may see an error similar to the one above.
+
+### Are you using the corect user?
+
+Different distributions are configured with different defaults users. See here the up to date list of [images](/cloud/pouta/images/#images) and their corresponding default users.
+
+It is a common practise for Pouta images, when you try to login as `root`, to get a message back telling you which username to use instead:
+
+```sh
+$ ssh root@86.xxx.xxx.xxx               
+Please login as the user "cloud-user" rather than the user "root".
+```
+
+### Is a key pair configured in the Instance?
+
+It can happen that when the instance was created you forgot to add a key pair during the creation. This can be checked from the [instance page](https://pouta.csc.fi/dashboard/project/instances/).
+
+![Instance with no Key Pair](/cloud/img/instance-no-keypair.png)
```

Comment:
No image are displayed. I think the correct path would be `../../cloud/img/instance-no-keypair.png`

Reply:
fixed